components/WhyMeditate/AudioWithSubs.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import subs from './subs.json';
+import styles from './AudioWithSubs.module.css';
+import cx from 'classnames';
+
+function findSegments(ms) {
+  const segments = subs.filter(
+    (s) => s.startMs > 0 && s.startMs < ms && s.startMs + s.duration > ms
+  );
+  return segments.map((segment) => {
+    const segmentStart = segment.startMs;
+    const wordOffset = ms - segmentStart;
+
+    return {
+      parts: segment.parts
+        ?.filter((s) => (s.delay || 0) <= wordOffset)
+        .map((part) => part.phrase),
+      key: segment.key,
+    };
+  });
+}
+
+export function AudioWithSubs({ time }) {
+  const specialWords = [
+    'reality',
+    'confuse',
+    'meditation',
+    'world',
+    'symbols',
+    'minds',
+    'wealth',
+    'images',
+    'dangerous',
+    'myself',
+    'think',
+    'madness',
+    'compulsively',
+  ];
+
+  return findSegments(time * 1000).map((segment, index) => {
+    const allWords = segment.parts.reduce((words, word) => (words += word), '');
+    const specialWord = segment.parts.find((word) =>
+      specialWords.includes(word.trim())
+    );
```

Comment:
I'm not a big fun of this effect anymore. Let's get rid of big special words and only keep the smaller ones.